engine/src/main/battlecode/common/RobotType.java

```diff
@@ -55,15 +55,15 @@ public enum RobotType {
      * 
      * @battlecode.doc.robottype
      */
-    BOOSTER         (  0,   0, 150, 140,  25, 40,   0, -1, 20, 10000),
+    BOOSTER         (  0,   0, 150, 140,  25, 400,   0, -1, 20, 10000),
     //               BCA  BCM  BCE   AC   MC  HP  DMG  AR  VR     BL
     /**
      * Amplifiers are the key to communication for your army. They enable 
      * robots within a range to write messages to their faction’s shared array.
      *
      * @battlecode.doc.robottype
      */
-    AMPLIFIER       ( 40,  40,   0,   -1,  15, 40,   0, -1, 34,  7500)
```

Comment:
looks like amplifier health used to be 40, not 400 (we should change this in the specs changelog)

Reply:
yeah again multiply by 10 is null op but feel free to change if you think it makes more sense